addons/l10n_ec/data/account.fiscal.position.csv

```diff
@@ -0,0 +1,8 @@
+id,name,active
+cont_especial,CONTRIBUYENTE ESPECIAL,True
```

Comment:
OK for templates, but the mapping is company dependant, there is no general rules, it is a case by case type of thing in ecuador.

Reply:
It seems weird for the user to supply him with empty fiscal positions. Then better to define nothing.

Reply:
There are several customizations by different partners on how to automate the taxes for withholds and so on, so it might be a good idea to left the mapping for a higher module, but ensure that at least a minimal list of fiscal positions is created even if it is empty.